app/models/study_file.rb

```diff
@@ -653,10 +655,36 @@ def local_location
     end
   end
 
+  # generate a download path to use with bulk_download
```

Comment:
move to BulkDownloadService, and have it take the study as an argument

Reply:
I need this here to use in other places (like views and responses)